app/uk/gov/hmrc/selfservicetimetopay/controllers/SelfServiceTimeToPayController.scala

```diff
@@ -34,6 +39,25 @@ class SelfServiceTimeToPayController extends TimeToPayController {
   def submit: Action[AnyContent] = Action { implicit request =>
     Redirect(routes.EligibilityController.start())
   }
+  def getSignInQuestion: Action[AnyContent] = Action.async { implicit request =>
+    sessionCache.get.map {
+      case Some(ttpData@TTPSubmission(_, _, _, _, typeOfTax@Some(_), _, _, _, _)) =>
```

Comment:
Navigation rules are not properly applied here. Type of tax is a tuple where both values need to be matched (true, false) and existingTTP also need to be matched against Some(false)